lang/java/reef-runtime-azbatch/src/main/java/org/apache/reef/runime/azbatch/client/AzureBatchRuntimeConfigurationStatic.java

```diff
@@ -43,5 +45,6 @@
       .bindImplementation(DriverConfigurationProvider.class, AzureBatchDriverConfigurationProviderImpl.class)
       .bindImplementation(RuntimeClasspathProvider.class, AzureBatchClasspathProvider.class)
       .bindImplementation(RuntimePathProvider.class, AzureBatchJVMPathProvider.class)
+      .bindImplementation(CommandBuilder.class, LinuxCommandBuilder.class)
```

Comment:
Adding this may break Window runtime. You may want to verify it. This commandbuilder class is specified in avro file.

Reply:
Fixed by adding a configuration value called IsWindows instead.